### DISCUSSION TITLE
20231031-inflearn-subtitle/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#44] [Comments]
# 20231031-inflearn-subtitle/

https://tech.inflab.com/20231031-inflearn-subtitle/

**Comment by gohsue-inflab** [comment]
선비 너무 멋있어요

**Comment by richinhim** [comment]
인프런 깃에 관련 내용이나 소스가 있을까요?

**Comment by jkf87** [comment]
와 멋지네요 
실제 서비스 구현까지 많은 고민과 시행착오가 담겨있네요 

저는 whisper-jax 로 영어 srt를 만들고 그걸 deepl로 한글srt를 만들어서 처리해서 개인적으로 활용하고 있는데요. 

이걸 웹 서비스로 만들어 보고싶었는데
큰 도움이 될것 같네요! 
감사합니다

**Comment by kiwini0K6ng** [comment]
와... 이 기능 완전 유용하게 쓰고 있는데 이렇게 탄생 배경과 과정까지 상세히 공유해주시고 감사합니다.
